src/dotty/tools/dotc/Compiler.scala

```diff
@@ -55,7 +55,8 @@ class Compiler {
            new Literalize,
            new Getters,
            new ElimByName,
-           new ResolveSuper),
+           new ResolveSuper,
+           new ClassTags),
```

Comment:
ClassTags could go anywhere after `ExtensionMethods`. It could even be in the same group.
I'm fine with moving it.
Can you explain why you want `ResolveSuper - erasure - Mixin` to be same unit? We should not rely on this, as there are phases that want to go in between(think class specialization, which changes superclasses)